lib/bojangles.rb

```diff
@@ -65,13 +65,15 @@ def get_avail_departure_times!
 
   def go!
     error_messages, statuses = DepartureComparator.compare
-
+    time_occurred = Time.now
+    
     if error_messages.present?
       MAIL_SETTINGS[:html_body] = message_html(error_messages)
       if CONFIG['environment'] == 'development'
         MAIL_SETTINGS.merge! via: :smtp, via_options: { address: 'localhost', port: 1025 }
       end
       Pony.mail MAIL_SETTINGS
+      update_log_file! to: time_occurred, error_messages, statuses
```

Comment:
These method arguments don't look right to me. In particular, I see three arguments:

1. `{ :to => Time.now }`;
2. `error_messages`,
3. `statuses`

But the update_log_file method only takes one argument. Am I missing something?